zeebe/atomix/cluster/src/main/java/io/atomix/cluster/messaging/MessagingConfig.java

```diff
@@ -35,6 +36,8 @@ public class MessagingConfig implements Config {
   private CompressionAlgorithm compressionAlgorithm = CompressionAlgorithm.NONE;
   private File keyStore;
   private String keyStorePassword;
+  private DataSize brokerSoSndbuf = DataSize.ofMegabytes(1);
```

Comment:
:question:  is this default needed? I see that you added default values in Broker config. I would avoid having too many defaults in different parts of the code base

Reply:
It prevents the NPE that occurs during Broker startup as we're retrieving the new parameters values and they're null which oddly seems to occurs on the second time `bootstrapServer` is invoked.

Reply:
Does it happen in unit tests or in qa tests?

Reply:
In alternative, it seems we can set the value of the parameters in the other instance of `MessagingConfig`.